src/editor.rs

```diff
@@ -387,6 +388,15 @@ impl Editor {
         }
     }
 
+    fn delete_current_row(&mut self) {
+        if self.cursor.y < self.rows.len() {
+            self.rows[self.cursor.y].chars.clear();
+            self.update_row(self.cursor.y, false);
+            self.cursor.x = 0;
+            self.delete_char();
```

Comment:
It looks like the first row is not removed when running the command.

Before:
```
1 │a
2 │b
3 │c
~ │
```

After:
```
1 │
2 │b
3 │c
~ │
```

What do you think of adding `self.cursor.y += 1;` before `self.delete_char();`? I think that would solve the problem